traffic_ops/traffic_ops_golang/api/api.go

```diff
@@ -369,6 +371,33 @@ func (inf *APIInfo) Close() {
 	}
 }
 
+// SendMail is a convenience method used to call SendMail using an APIInfo structure's configuration.
+func (inf *APIInfo) SendMail(to rfc.EmailAddress, msg []byte) (int, error, error) {
+	return SendMail(to, msg, inf.Config)
+}
+
+// SendMail sends an email msg to the address identified by to. The msg parameter should be an
+// RFC822-style email with headers first, a blank line, and then the message body. The lines of msg
+// should be CRLF terminated. The msg headers should usually include fields such as "From", "To",
+// "Subject", and "Cc". Sending "Bcc" messages is accomplished by including an email address in the
+// to parameter but not including it in the msg headers.
+// The cfg parameter is used to set things like the "From" field, as well as for connection
+// and authentication with an external SMTP server.
+// SendMail returns (in order) an HTTP status code, a user-friendly error, and an error fit for
+// logging to system error logs. If either the user or system error is non-nil, the operation failed,
+// and the HTTP status code indicates the type of failure.
+func SendMail(to rfc.EmailAddress, msg []byte, cfg *config.Config) (int, error, error) {
+	if !cfg.SMTP.Enabled {
+		return http.StatusServiceUnavailable, errors.New("SMTP is not enabled!"), nil
```

Comment:
Nah, I don't think so. There's (presumably) nothing wrong with the request, it's just that the server is incapable of answering the request because of an absentee service.

Reply:
is the configuration options something the user needs to do through params though? If the user can take action and fix it then should it be 4xx? idk might be overthinking it 🤷‍♂

Reply:
Yeah, this should be a 5xx.  It's not about whether the user can do something to fix it, it's about who's fault it is. HTTP says 4xx are for errors that are the user's fault, 5xx if its the server's fault. 

See https://tools.ietf.org/html/rfc7231#section-6